packages/context/src/binding.ts

```diff
@@ -31,16 +32,19 @@ export class Binding {
   }
 
   to(value: BoundValue): this {
-    this.getValue = () => value;
+    // tslint:disable-next-line:promise-function-async
+    this.getValue = () => Promise.resolve(value);
     return this;
   }
 
   toDynamicValue(factoryFn: () => BoundValue): this {
-    this.getValue = factoryFn;
+    // tslint:disable-next-line:promise-function-async
```

Comment:
I proposed a feature for `tslint` that would make this comment unnecessary - see I proposed a feature for `tslint` that would make this